pyproject.toml

```diff
@@ -291,6 +291,13 @@ dl = [
   'transformers[torch]<4.41.0; python_version < "3.12"',
   'pykan<0.2.2,>=0.2; python_version > "3.9.7"',
   'pytorch-forecasting>=1.0.0; python_version < "3.11"',
+  'lightning>=2.0; python_version < "3.12"',
+  'hydra-core>=1.3; python_version < "3.12"',
```

Comment:
Can you tell me where this is required? I haven't seen any usage of hydra-core in the vendored code.